tg_bot/modules/bans.py

```diff
@@ -191,6 +191,82 @@ def unban(bot: Bot, update: Update, args: List[str]) -> str:
     return log
 
 
+@run_async
+@bot_admin
+def rban(bot: Bot, update: Update, args: List[str]):
+    message = update.effective_message
+
+    if not args:
+        message.reply_text("You don't seem to be referring to a chat/user.")
+        return
+
+    user_id, chat_id = extract_user_and_text(message, args)
+
+    if not user_id:
+        message.reply_text("You don't seem to be referring to a user.")
+        return
+    elif not chat_id:
+        message.reply_text("You don't seem to be referring to a chat.")
+        return
+
+    try:
+        chat = bot.get_chat(chat_id)
```

Comment:
Small nitpick - if people write _after_ the chat_id, that text will be included. how about .split() and take [0]? avoid as many errors as possible, as early as possible